README.md

```diff
@@ -6,76 +6,76 @@
 [![Python](https://img.shields.io/badge/python-2.7%20%7C%203.6-blue?logo=python)](https://kodi.tv/article/attention-addon-developers-migration-python-3)
 [![Transifex](https://img.shields.io/badge/transifex-en|nl|sv|no|fi-blueviolet)](https://www.transifex.com/retrospect/public/)
 
-This repository holds the main code for Retrospect. For more information on bug reporting, please visit https://github.com/retrospect-addon/plugin.video.retrospect or https://www.rieter.net/content/.
+This repository holds the main code for Retrospect. For more info on reporting bugs, please visit https://github.com/retrospect-addon/plugin.video.retrospect or https://www.rieter.net/content/.
 
 # Introduction #
 Retrospect (Previously XBMC Online TV) is a Kodi video add-on which allows you to watch streams of 
-a number of free and publicly available online TV stream sites.
+a number of gratis publicly available online TV stream sites.
```

Comment:
I would keep "free and publicly", as "gratis" seems so old fashion.

Reply:
Changed it around to be less awkward. The problem with "free" is that it may be confused for "Free Software TV streaming sites".

Reply:
Didn't see this, sorry. As I commented below, I suggest:
"allows you to watch a number of freely available online TV streaming sites"

Gratis is indeed archaic. Why "free" would cause that confusion I don't understand, @comradekingu.

Also, remove "(Previously XBMC Online TV)".